### PR TITLE
lnd: init at 0.7.0-beta

### PR DESCRIPTION
**Pull request by stefan-mihaila**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1148,6 +1148,19 @@
...
+    keys = [
+      { longkeyid = "rsa4096/6E68A39BF16A3ECB";
+        fingerprint = "CBC9 C7CC 51F0 4A61 3901 C723 6E68 A39B F16A 3ECB";
+      }
+      { longkeyid = "rsa4096/6220AD7846220A52";
+        fingerprint = "7EAB 1447 5BBA 7DDE 7092 7276 6220 AD78 4622 0A52";
+      }
+    ];
+  };
   dalance = {
```
Comment (kalbasit) [inline maintainers/maintainer-list.nix]: You should have this with a separate commit (same pr).

File: pkgs/applications/altcoins/lnd.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  name = "lnd-${version}";
```
Comment (kalbasit) [inline pkgs/applications/altcoins/lnd.nix]:
```suggestion
  pname = "lnd";
```

File: pkgs/applications/altcoins/lnd.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  modSha256 = "0akxi7xhyz7xx0vc003abidva02sp940cc2gfjg4fmzkc95cajc9";
+
+  meta = with lib; {
+    description = "Lightning Network Daemon";
+    homepage = https://github.com/lightningnetwork/lnd;
+    license = lib.licenses.mit;
+    maintainers = with maintainers; [ cypherpunk2140 ];
+    platforms = platforms.unix;
```
Comment (kalbasit) [inline pkgs/applications/altcoins/lnd.nix]: Is this necessary? `buildGoModule` defines it to platforms.all if found undefined.

File: pkgs/applications/altcoins/lnd.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    repo = "lnd";
+    rev = "v${version}";
+    sha256 = "0d6m1vfy33rg6d7qmkpydiypav1girxsnxan9njyjz0vhinmq0sx";
+  };
+
+  modSha256 = "0akxi7xhyz7xx0vc003abidva02sp940cc2gfjg4fmzkc95cajc9";
+
+  meta = with lib; {
+    description = "Lightning Network Daemon";
+    homepage = https://github.com/lightningnetwork/lnd;
```
Comment (kalbasit) [inline pkgs/applications/altcoins/lnd.nix]: Please put the URL in quotes.

**Review by kalbasit — CHANGES_REQUESTED** [review]

File: pkgs/applications/altcoins/lnd.nix (hunk 3 above)
Comment (cypherpunk2140) [reply pkgs/applications/altcoins/lnd.nix]: I was under the impression `lnd` doesn't work on Windows, but I was wrong, good catch.

**Comment by kalbasit** [comment]
@GrahamcOfBorg build lnd

**Review by kalbasit — APPROVED** [review]

**Pull request merged**